programs/ssl/ssl_client2.c

```diff
@@ -1466,6 +1546,47 @@ int main( int argc, char *argv[] )
 #endif
 
 #if defined(MBEDTLS_KEY_EXCHANGE__SOME__PSK_ENABLED)
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+    if( opt.psk_slot != 0 )
+    {
+        status = psa_crypto_init();
```

Comment:
Note: this should be removed when rebasing on the PR that adds it near the beginning of `main()`.